tools/doc/html.js

```diff
@@ -188,6 +195,49 @@ function replaceInText(text) {
   return linkJsTypeDocs(linkManPages(text));
 }
 
+function altDocs(filename) {
+  let html = '';
+
+  if (!docCreated) {
+    console.error('Failed to add alternative version links');
+    return html;
+  }
+
+  function lte(v) {
```

Comment:
String based comparison with `v.num` is not good enough. Why not split it at dots and the do integer comparison?